src/components/itemMediaInfo/itemMediaInfo.js

```diff
@@ -145,13 +146,13 @@ function getMediaSourceHtml(user, item, version) {
         if (stream.BitDepth) {
             attributes.push(createAttribute(globalize.translate('MediaInfoBitDepth'), `${stream.BitDepth} bit`));
         }
-        if (stream.VideoRange) {
+        if (stream.VideoRange && stream.Type === 'Video') {
             attributes.push(createAttribute(globalize.translate('MediaInfoVideoRange'), stream.VideoRange));
         }
-        if (stream.VideoRangeType) {
+        if (stream.VideoRangeType && stream.Type === 'Video') {
             attributes.push(createAttribute(globalize.translate('MediaInfoVideoRangeType'), stream.VideoRangeType));
         }
-        if (stream.VideoDoViTitle) {
+        if (stream.VideoDoViTitle && stream.Type === 'Video') {
```

Comment:
`VideoDoViTitle` is [`undefined`](https://github.com/jellyfin/jellyfin/blob/b1870792b1310b58f84089c1c50deab1b7d12f24/MediaBrowser.Model/Entities/MediaStream.cs#L214) for non-video streams. So I think it is safe not to check the stream type.
```suggestion
        if (stream.VideoDoViTitle) {
```